packages/halo-theme/src/custom-elements/ef-pill.less

```diff
@@ -36,6 +36,10 @@
     }
   }
 
+  &[focused="visible"] {
+    border: 1px solid @scheme-color-primary;
+  }
```

Comment:
Why do you need this ? I think pill style should be inherited from elemental without the need to add another line of style.